doc/policies/admin.rst

```diff
@@ -705,42 +705,42 @@ is required.
 
 New in version 3.5.
 
-.. _admin_set_user_attributes:
+.. _admin_set_custom_user_attributes:
 
-set_user_attributes
-~~~~~~~~~~~~~~~~~~~~
+set_custom_user_attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 type: string
 
 New in version 3.6
 
-This policy defines which additional attributes an adminsitrator is allowed to set.
+This policy defines which additional attributes an administrator is allowed to set.
 It can also define, to which value the admin is allowed to set such attribute.
 For allowing all values, the asterisk ("*") is used.
 
 .. note:: That commata are not allowed in policy actions value, so the setting has to
    be define by separating colons (":") and spaces.
 
-Each colon separates the key from the allowed values. Values in a list a separated by spaces.
+Each key is enclosed in colons and followed by a list of values separated by spaces.
```

Comment:
Note explicitly, that this of course means that there are not whitespaces allowed in each of the values.

Reply:
will do

Reply:
Done in 9afc7703